data-prepper-plugins/opensearch/src/main/resources/metrics-otel-v1-index-template.json

```diff
@@ -0,0 +1,241 @@
+{
+  "version": 1,
```

Comment:
See comments from the metrics template above:
* float -> double
* integer -> long
* text -> keyword

Reply:
See replies